src/devices/Bmxx80/Bmx280Base.cs

```diff
@@ -66,9 +84,12 @@ public FilteringMode ReadFilterMode()
         /// <returns>Calculated temperature.</returns>
         public async Task<Temperature> ReadTemperatureAsync()
         {
+            if (TemperatureSampling == Sampling.Skipped)
```

Comment:
I guess I've never thought of it through with the sampling, but is there any scenario where somebody that have Sampling.Skipped set, and would call this method? I guess I don't understand why would somebody call this method when sampling is skipped, but it might be that they actually care about reading the temperature even when sampling was disabled.

Reply:
My thought with this was that it's just the most unambiguous way to indicate that no measurement was taken. There is probably less room for error if the method returns NaN instead of the last value measured so an out of date measurement is not mistakenly used.

The scenario where someone wants to read the last measured value even if its from many cycles ago seems unlikely to me since you would probably just log them when they are created.

Reply:
![Capture](https://user-images.githubusercontent.com/27669180/63122904-15a42280-bfa8-11e9-8850-ceaedceed76d.PNG)

Actually reading the registers after the measurements are skipped just returns wrong data so in case we want to return the last measurement we would have to save that through a field too.